pkg/plugins/platform/utils.go

```diff
@@ -208,3 +214,79 @@ func ddevContainersRunning() (bool, error) {
 	}
 	return false, nil
 }
+
+// SetOfflineMode enables offline mode for ddev. When offline mode is
+// set, a tracking file is created and hosts entries are created for
```

Comment:
Say ~/.ddev/.offline instead of "a tracking file"

Reply:
Should we reverse the logic and make it offline by default? So maybe make a file named .usedns?